docker-compose-https.yml

```diff
@@ -19,14 +22,14 @@ services:
       WEBLATE_ENABLE_HTTPS: 1
       WEBLATE_IP_PROXY_HEADER: HTTP_X_FORWARDED_FOR
   database:
-    image: postgres:16-alpine
+    image: postgres:16.2
```

Comment:
Why do you think pinning major version is not good enough here? We're not really playing on the exact environment here (Weblate itself is not pinned at all).

Also, I don't see a reason for stopping using alpine variant, which is smaller.

Reply:
> Why do you think pinning major version is not good enough here?

Because 16.2 is newer than 16!

Reply:
No, 16 is a moving tag for the latest release in 16.x series, see https://hub.docker.com/_/postgres.

And I also ask for removal of the `-alpine` suffix.

Reply:
> No, 16 is a moving tag for the latest release in 16.x series, see https://hub.docker.com/_/postgres.

So, the _16_ tag automatically takes the latest sub-version of the 16.x series?